src/shared/components/challenge-detail/Header/ChallengeTags.jsx

```diff
@@ -36,15 +36,15 @@ export default function ChallengeTags(props) {
   let EventTag;
   let TrackTag;
   switch (track) {
-    case COMPETITION_TRACKS.DATA_SCIENCE:
+    case COMPETITION_TRACKS.DS:
```

Comment:
@LieutenantRoger Please make sure to update all ref to `COMPETITION_TRACKS` like you have done here.

ref. comment here: https://github.com/topcoder-platform/topcoder-react-lib/commit/d7e336ea391ebf3c950cf09f999a48a00db7402a#r42801128